### PR TITLE
[GIT PULL] Add IORING_OP_SOCKET to man io_uring_enter

### PR DESCRIPTION
**Pull request by ilikdoge**
Document the IORING_OP_SOCKET opcode in io_uring_enter.2
----
## git request-pull output:
```
The following changes since commit 1c469ee303d0f458a6b039c9483865b3af248400:

  man/io_uring_prep_accept.3: add note on multishot and addr/addrlen (2022-10-06 15:32:19 -0600)

are available in the Git repository at:

  https://github.com/ilikdoge/liburing master

for you to fetch changes up to 09b542e6939b5fb112de9e0b696b024b464944c4:

  man/io_uring_enter.2: Add IORING_OP_SOCKET to the list of opcodes (2022-10-15 22:48:59 -0700)

----------------------------------------------------------------
ilikdoge (1):
      man/io_uring_enter.2: Add IORING_OP_SOCKET to the list of opcodes

 man/io_uring_enter.2 | 34 ++++++++++++++++++++++++++++++++++
 1 file changed, 34 insertions(+)
```
----
<details>
<summary>Click to show/hide pull request guidelines</summary>

## Pull Request Guidelines
1. To make everyone easily filter pull request from the email
notification, use `[GIT PULL]` as a prefix in your PR title.
```
[GIT PULL] Your Pull Request Title
```
2. Follow the commit message format rules below.
3. Follow the Linux kernel coding style (see: https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst).

### Commit message format rules:
1. The first line is title (don't be more than 72 chars if possible).
2. Then an empty line.
3. Then a description (may be omitted for truly trivial changes).
4. Then an empty line again (if it has a description).
5. Then a `Signed-off-by` tag with your real name and email. For example:
```
Signed-off-by: Foo Bar <foo.bar@gmail.com>
```

The description should be word-wrapped at 72 chars. Some things should
not be word-wrapped. They may be some kind of quoted text - long
compiler error messages, oops reports, Link, etc. (things that have a
certain specific format).

Note that all of this goes in the commit message, not in the pull
request text. The pull request text should introduce what this pull
request does, and each commit message should explain the rationale for
why that particular change was made. The git tree is canonical source
of truth, not github.

Each patch should do one thing, and one thing only. If you find yourself
writing an explanation for why a patch is fixing multiple issues, that's
a good indication that the change should be split into separate patches.

If the commit is a fix for an issue, add a `Fixes` tag with the issue
URL.

Don't use GitHub anonymous email like this as the commit author:
```
123456789+username@users.noreply.github.com
```

Use a real email address!

### Commit message example:
```
src/queue: don't flush SQ ring for new wait interface

If we have IORING_FEAT_EXT_ARG, then timeouts are done through the
syscall instead of by posting an internal timeout. This was done
to be both more efficient, but also to enable multi-threaded use
the wait side. If we touch the SQ state by flushing it, that isn't
safe without synchronization.

Fixes: https://github.com/axboe/liburing/issues/402
Signed-off-by: Jens Axboe <axboe@kernel.dk>
```

</details>

----
## By submitting this pull request, I acknowledge that:
1. I have followed the above pull request guidelines.
2. I have the rights to submit this work under the same license.
3. I agree to a Developer Certificate of Origin (see https://developercertificate.org for more information).

### CODE REVIEW

**Comment by ilikdoge** [comment]
Not sure which version it was added, put a question mark in the version

File: man/io_uring_enter.2 [hunk 1]
```diff
@@ -1059,6 +1059,40 @@ value being passed in. This request type can be used to either just wake or
...
+installation is guaranteed to not fail. If there is already a file in the slot,
+it will be replaced, similar to
+.B IORING_OP_FILES_UPDATE.
+Please note that only io_uring has access to such files and no other syscall
+can use them. See
+.B IOSQE_FIXED_FILE
+and
+.B IORING_REGISTER_FILES.
+
+Available since 5.1?.
```
Comment (axboe) [inline man/io_uring_enter.2:1094]: This should be 5.19.

Comment (ilikdoge) [reply man/io_uring_enter.2]: Fixed

**Comment by axboe** [comment]
Can you please fold them into one commit? There's no point in having a wrong statement, then a fixup commit. Should just be one good commit. Thanks!

**Comment by ilikdoge** [comment]
Still rather new to git, hope this fixes it

**Comment by axboe** [comment]
Looks good now, thanks!

**Pull request merged**